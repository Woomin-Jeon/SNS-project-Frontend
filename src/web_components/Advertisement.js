import advertising1 from "../advertising.png";
import advertising2 from "../advertising2.png";
import React from "react";

function Advertisement() {
  return (
    <div className="advertising">
      <div className="advertising-sponsored">Sponsored</div>
      <a className="advertising-1" href="https://woogod.netlify.com/">
        <img className="advertising1-image" src={advertising1} />
        <div className="advertising-url">
          programmer.co.kr
        </div>
        <div className="advertising-statement">
          리액트 핵 선배들이 알려주는 실무 꿀팁 가득한 스터디!
        </div>
      </a>
      <a className="advertising-2" href="https://woogod.netlify.com/">
        <img className="advertising2-image" src={advertising2} />
        <div className="advertising-url">
          programmer.co.kr
        </div>
        <div className="advertising-statement">
          2020 Dev Match
        </div>
      </a>
    </div>
  );
}

export default Advertisement;
