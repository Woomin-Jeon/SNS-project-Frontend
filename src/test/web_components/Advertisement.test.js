import Advertisement from "../../web_components/Advertisement";
import React from 'react';
import {shallow} from 'enzyme';

describe('Advertisement', () => {
  it('renders well', () => {
    const component = shallow(
      <Advertisement />
    );

    expect(component).toMatchSnapshot();
  });
});
