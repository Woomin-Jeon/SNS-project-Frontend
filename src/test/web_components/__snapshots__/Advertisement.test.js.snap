// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Advertisement renders well 1`] = `ShallowWrapper {}`;
